.github/workflows/images.yaml

```diff
@@ -0,0 +1,70 @@
+---
+name: Build and push operator, bundle and catalog images
+
+on:
+  repository_dispatch:
+    types:
+      - operand-image-update
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+jobs:
+  get-operator-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
+    steps:
+      - name: Identify version based on branch
+        id: get-version
+        run: |
+          if [ ${{ github.ref_name }} == "main" ]; then
+              version="99.0.0"
+          else
+              version="${{ github.ref_name }}"
+          fi
+          echo "::set-output name=version::${version}"
+
+  build-images:
+    needs: get-operator-version
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ needs.get-operator-version.outputs.version }}
+    steps:
+      - name: Checkout current repository for the Dockerfiles
+        uses: actions/checkout@v3
+
+      - name: Login to the container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:v${{ env.VERSION }}
+        id: build-push-operator
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: "${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:v${{ env.VERSION }}"
+
+      - name: Build and push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-bundle:v${{ env.VERSION }}
+        id: build-push-operator-bundle
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: bundle.Dockerfile
+          push: true
+          tags: "${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-bundle:v${{ env.VERSION }}"
```

Comment:
Why don't we use the make targets for building and pushing images? IMO it leaves place for potential discrepancies between the two methods.

Reply:
Building and pushing the OLM bundle container image is as plain as `docker build` and `docker push`, since we already have the [bundle.Dockerfile](https://github.com/empovit/fdo-operator/blob/main/bundle.Dockerfile) and [bundle/](https://github.com/empovit/fdo-operator/tree/main/bundle) availablte in the repository.

On the other hand, building and pushing the OLM catalog is not just running docker commands, since we need [OPM](https://github.com/empovit/fdo-operator/blob/main/Makefile#L257-L258) to build the catalog image. 

Nevertheless, I see your point and we could in fact replace the `build-push-operator-bundle` action with `make bundle-build` and `make bundle-push`. But it wouldn't change the fact that those 2 actions are not supposed to be of the same functionality. They may both build and push images, but their `build` actions are quite different.